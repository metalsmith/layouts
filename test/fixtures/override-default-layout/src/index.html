---
layout: override.hbs
---
<h1>The title</h1>
