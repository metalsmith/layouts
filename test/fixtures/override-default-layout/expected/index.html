<!DOCTYPE html>
<html>
  <head>
    <title>Overridden template</title>
  </head>
  <body>
    <h1>The title</h1>

  </body>
</html>
